controllers/add_services.go

```diff
@@ -67,36 +67,26 @@ func (a addServices) reconcile(ctx context.Context, r *FoundationDBClusterReconc
 				continue
 			}
 
-			idNum, err := processGroup.ProcessGroupID.GetIDNumber()
-			if err != nil {
-				return &requeue{curError: err}
-			}
-
-			serviceName, _ := cluster.GetProcessGroupID(processGroup.ProcessClass, idNum)
-			service, err := internal.GetService(cluster, processGroup.ProcessClass, idNum)
+			service, err := internal.GetService(cluster, processGroup)
 			if err != nil {
-				return &requeue{curError: err}
+				return &requeue{curError: err, delayedRequeue: true}
```

Comment:
What is the "delayedRequeue" about? It appears you are setting it in all "require()" calls now.